.eslintrc.js

```diff
@@ -37,10 +37,16 @@ module.exports = {
       "always",
       { exceptAfterSingleLine: true },
     ],
-    "jsx-a11y/label-has-associated-control": ['error', {
-      "assert":"either"
-    }],
-    "react/static-property-placement": ["error", "static public field"]
+    "jsx-a11y/label-has-associated-control": [
+      "error",
+      {
+        assert: "either",
+      },
+    ],
+    "react/static-property-placement": "off",
+    "class-methods-use-this": "off",
+    "react/no-unused-class-component-methods": "off",
+    "default-param-last": "off",
```

Comment:
We can revisit this if we wish at a later point, but I'm not too pressed.
It would require rewriting a fair amount of APIService methods and their use across the front-end codebase.
IMO with typescript and a decent integration in your IDE this isn't required.